docs/modules/ROOT/pages/how-tos/upgrade-mimir-v1-v2.adoc

```diff
@@ -0,0 +1,237 @@
++
+[source,bash]
+----
+cd inventory/classes/$TENANT_ID
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.alertmanager.zoneAwareReplication.enabled = false" ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.ingester.zoneAwareReplication.enabled = false" ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.store_gateway.zoneAwareReplication.enabled = false" ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.rollout_operator.enabled = false" ${CLUSTER_ID}.yml
+
+cd ../../..
+----
++
+This configuration disables zone aware replication. In the first step, the new component version is deployed with this setting disabled.
+
+. Compile and push the cluster catalog
+. Wait for all relevant pods to start and become ready
+
+== Migrate Alertmanager
+
+. Update the confiugration for alertmanager
++
+[source,bash]
+----
+cd inventory/classes/$TENANT_ID
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.alertmanager.zoneAwareReplication.enabled = true" ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.alertmanager.zoneAwareReplication.migration.enabled = true" ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.rollout_operator.enabled = true" ${CLUSTER_ID}.yml
+
+cd ../../..
+----
+
+. Compile and push the cluster catalog
+. Wait for all alertmanager pods to restart and become ready
+. Once that's done, enable the write path for the new zone aware alertmanager:
++
+[source,bash]
+----
+cd inventory/classes/$TENANT_ID
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.alertmanager.zoneAwareReplication.migration.writePath = true" ${CLUSTER_ID}.yml
+
+cd ../../..
+----
+
+. Compile and push the cluster catalog
+. Wait for the new zone-aware alertmanager pods to become ready
+
+. Next, set the final configuration (meaning the default configuration) by removing the custom `alertmanager` config block:
++
+[source,bash]
+----
+cd inventory/classes/$TENANT_ID
+
+yq eval -i "del(.parameters.${MIMIR_PARAM_KEY}.helm_values.alertmanager)" ${CLUSTER_ID}.yml
+
+cd ../../..
+----
+. Compile and push the cluster catalog
+. Wait for the old zone-unaware alertmanager pods to be terminated
+
+== Migrate store-gateway
+
+. Scale down the store-gateways
++
+[source,bash]
+----
+cd inventory/classes/$TENANT_ID
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.store_gateway.replicas = 0" ${CLUSTER_ID}.yml
+
+cd ../../..
+----
+
+. Compile and push the cluster catalog
+. Wait for all store-gateway pods to terminate
+
+. Next, set the final configuration (meaning the default configuration) by removing the custom `store_gateway` config block:
++
+[source,bash]
+----
+cd inventory/classes/$TENANT_ID
+
+yq eval -i "del(.parameters.${MIMIR_PARAM_KEY}.helm_values.store_gateway)" ${CLUSTER_ID}.yml
+
+cd ../../..
+----
+. Compile and push the cluster catalog
+. Wait for the new store-gwateways to become ready
+
+== Migrate ingesters
```

Comment:
I didn't figure out the details here, but some of the catalog compilations can probably be replaced by disabling argocd autosync and `kubectl scale`